### PR TITLE
added sim page to start and run the game

### PR DESCRIPTION
**Pull request by kaylamuraoka**
added sim page to start and run the game

### CODE REVIEW

**Review by kaylamuraoka — COMMENTED** [review]
request pull request

**Review by stephimarie — COMMENTED** [review]
Looks good

**Review by stephimarie — APPROVED** [review]

File: config/config.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    password: null,
+    database: "database_test",
+    host: "127.0.0.1",
+    dialect: "mysql",
+  },
+  production: {
+    use_env_variable: "JAWSDB_URL",
+    dialect: "mysql",
+  },
+};
```
Comment (stephimarie) [inline config/config.js:20]: good

File: models/index.js [hunk 2]
```diff
@@ -1,31 +1,41 @@
-'use strict';
+"use strict";
```
Comment (stephimarie) [inline models/index.js:1]: good

File: models/index.js [hunk 3]
```diff
@@ -1,31 +1,41 @@
...
 
-const fs = require('fs');
-const path = require('path');
-const Sequelize = require('sequelize');
+const fs = require("fs");
+const path = require("path");
+const Sequelize = require("sequelize");
```
Comment (stephimarie) [inline models/index.js:5]: good

File: models/index.js [hunk 4]
```diff
@@ -1,31 +1,41 @@
...
 const basename = path.basename(__filename);
-const env = process.env.NODE_ENV || 'development';
-const config = require(__dirname + '/../config/config.json')[env];
+const env = process.env.NODE_ENV || "development";
+const config = require(__dirname + "/../config/config.js")[env];
+// DB setup with JawsDB for Heroku
```
Comment (stephimarie) [inline models/index.js:9]: good

File: models/index.js [hunk 5]
```diff
@@ -1,31 +1,41 @@
...
 if (config.use_env_variable) {
   sequelize = new Sequelize(process.env[config.use_env_variable], config);
 } else {
-  sequelize = new Sequelize(config.database, config.username, config.password, config);
+  sequelize = new Sequelize(
+    config.database,
+    config.username,
+    config.password,
+    config
+  );
```
Comment (stephimarie) [inline models/index.js:21]: good

File: models/index.js [hunk 6]
```diff
@@ -1,31 +1,41 @@
...
 
-fs
-  .readdirSync(__dirname)
-  .filter(file => {
-    return (file.indexOf('.') !== 0) && (file !== basename) && (file.slice(-3) === '.js');
+fs.readdirSync(__dirname)
+  .filter((file) => {
+    return (
+      file.indexOf(".") !== 0 && file !== basename && file.slice(-3) === ".js"
+    );
```
Comment (stephimarie) [inline models/index.js:28]: good

File: models/index.js [hunk 7]
```diff
@@ -1,31 +1,41 @@
...
   })
-  .forEach(file => {
-    const model = require(path.join(__dirname, file))(sequelize, Sequelize.DataTypes);
+  .forEach((file) => {
+    const model = require(path.join(__dirname, file))(
+      sequelize,
+      Sequelize.DataTypes
+    );
     db[model.name] = model;
```
Comment (stephimarie) [inline models/index.js:35]: good

File: models/index.js [hunk 8]
```diff
@@ -1,31 +1,41 @@
...
   });
 
-Object.keys(db).forEach(modelName => {
+Object.keys(db).forEach((modelName) => {
```
Comment (stephimarie) [inline models/index.js:38]: good

File: package.json [hunk 9]
```diff
@@ -5,19 +5,19 @@
   "main": "server.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "start": "node server.js"
+    "start": "npm start"
```
Comment (stephimarie) [inline package.json:8]: good

File: package.json [hunk 10]
```diff
@@ -5,19 +5,19 @@
...
   },
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/VictorMoscone/inhumans.git"
+    "url": "git+https://github.com/kaylamuraoka/inhumans"
```
Comment (stephimarie) [inline package.json:12]: good

File: package.json [hunk 11]
```diff
@@ -5,19 +5,19 @@
...
   },
   "keywords": [],
   "author": "",
   "license": "ISC",
   "bugs": {
-    "url": "https://github.com/VictorMoscone/inhumans/issues"
+    "url": "https://github.com/kaylamuraoka/inhumans/issues"
```
Comment (stephimarie) [inline package.json:18]: good

File: package.json [hunk 12]
```diff
@@ -5,19 +5,19 @@
...
   },
-  "homepage": "https://github.com/VictorMoscone/inhumans#readme",
+  "homepage": "https://github.com/kaylamuraoka/inhumans#readme",
```
Comment (stephimarie) [inline package.json:20]: good

**Review by stephimarie — APPROVED** [review]

**Review by lstvgore — APPROVED** [review]

**Pull request closed without merging**